src/config/dynamic-theme-fixes.config

```diff
@@ -996,8 +996,24 @@ CSS
 use.valueCircle_f6otssy {
     stroke: url(#pb-gradient-0);
 }
+:root {
+    --selecetdHighlightOpacity: 0.2;
```

Comment:
It's a constant, so let's remove this variable and just implement it onto the rgb itself.

Reply:
```suggestion
```